cli/start.go

```diff
@@ -78,7 +79,23 @@ func start(ctx context.Context, containerID string) (vc.VCSandbox, error) {
 	}
 
 	if containerType.IsSandbox() {
-		return vci.StartSandbox(sandboxID)
+		s, err := vci.StartSandbox(sandboxID)
+		if err != nil {
+			return nil, err
+		}
+
+		ociSpec, err := oci.GetOCIConfig(status)
+		if err != nil {
+			return nil, err
+		}
+
+		if err := enterNetNS(s.GetNetNs(), func() error {
+			return postStartHooks(ociSpec, sandboxID, status.Annotations[vcAnnot.BundlePathKey])
+		}); err != nil {
```

Comment:
Same comment as above - this is compact, but rather difficult to understand ;)

Reply:
done